site/styles/_meta-detail.scss

```diff
@@ -590,3 +576,15 @@
     font-size: 12px;
   }
 }
+
+s72-element-switcher {
+  @extend .mb-5;
```

Comment:
Can you replace this with `margin-bottom`... `mb-5` is a one liner anyway so we don't gain anything by obfuscating.

Reply:
- replaced